docs/components/toaster.md

```diff
@@ -2,63 +2,61 @@
 
 KToaster a popup notification component. The toaster will close automatically after a set timeout or can be dismissed by clicking on the close icon.
 
-KToaster is used via the `ToastManager` instance. All rendering is controlled from ToastManager via an intuitive, imperative API. It is recommended that you initialize `ToastManager` in your app via [`app.config.globalProperties`](https://vuejs.org/api/application.html#app-config-globalproperties) to allow you to access it on any component instance inside your application.
+KToaster is used via the `ToastManager` instance. All rendering is controlled from `ToastManager` via an intuitive API.
+
+The easiest way to use it is by creating a composable that you can use anywhere in your app. This way you don't have to initialize `ToastManager` in every component.
 
 ```ts
-// main.ts
+// composables/useToaster
 
-import { createApp } from 'vue'
+import { getCurrentInstance, onBeforeUnmount } from 'vue'
```

Comment:
issue: we shouldn't be using `getCurrentInstance` as it's a Vue internal and not recommended for normal use (the API may change, or it may become unavailable in future releases).

References:
- https://github.com/vuejs/vue/blob/main/src/v3/currentInstance.ts
- https://github.com/vuejs/docs/issues/1422#issuecomment-1032120675

Reply:
I've tried and it seems like appContext isn't required for it to work. Let me try and see if we can get away with not having to rely on `getCurrentInstance `.